src/runtime/components/CookieControl.vue

```diff
@@ -141,7 +141,13 @@
                                 :key="entry[0]"
                               >
                                 <br />
-                                <a :href="entry[0]">{{
+                                <NuxtLink
+                                  v-if="!entry[0].startsWith('http')"
+                                  :to="entry[0]"
+                                  @click="isModalActive = false"
+                                  >{{ entry[1] || entry[0] }}</NuxtLink
+                                >
+                                <a v-else :href="entry[0]">{{
```

Comment:
I think this `else` can be replaced by [NuxtLink's](https://nuxt.com/docs/api/components/nuxt-link#nuxtlink) `external` property.

Reply:
updated the NuxtLink to use external

Reply:
Please let me mark my comments as resolved once I confirmed the new implementation matches what I was thinking of :pray:

Reply:
oops - apologies